roxctl/common/flags/endpoint.go

```diff
@@ -108,6 +109,10 @@ func EndpointAndPlaintextSetting() (string, bool, error) {
 		}
 	}
 
+	if _, _, err := net.SplitHostPort(u.Host); err != nil {
+		return "", false, errox.InvalidArgs.Newf("invalid endpoint: %s, the scheme should be: http(s)://<endpoint>:<port>", err.Error())
```

Comment:
UX: port must be optional if scheme is specified.

Reply:
Consider also refactoring `CentralURL()` below in this file. Looks like we reassemble the URL ignoring the provided scheme. I think that we shouldn't do any check of the provided endpoint, and fail only when we really try to connect.

Reply:
If port is not given, but must be provided to some function, we should put the default for the scheme.

Reply:
added default port